TASVideos/Pages/Forum/Subforum/Index.cshtml

```diff
@@ -11,8 +11,12 @@
 }
 
 <partial Name="_ForumHeader" />
-<a permission="EditForums" asp-page="Edit" asp-route-id="@Model.Forum.Id" class="float-end btn btn-primary btn-sm border border-warning"><i class="fa fa-pencil"></i> Edit</a>
-<a permission="CreateForumTopics" asp-page="/Forum/Topics/Create" asp-route-forumId="@Model.Forum.Id" class="me-2 float-end btn btn-primary btn-sm"><span class="fa fa-plus"></span> New Topic</a>
+<div class="row">
```

Comment:
you can use <full-row> and it does both these row and col div classes for you

Reply:
use what? I don't get what you mean